src/cljs/rems/spa.cljs

```diff
@@ -648,9 +620,39 @@
 
 ;;;; Initialize app
 
+(defn- lazy-load-data!
+  "Loads datasets that are not required for immediate render or e.g. require login."
+  []
+  (let [user @(rf/subscribe [:user])
+        organizations @(rf/subscribe [:organizations])]
+    (when (and user (empty? organizations))
+      (config/fetch-organizations!))))
+
+(defn- grab-focus!
+  "Helper that sets focus on prominent header element when user navigates to new page."
+  []
+  (when (and @(rf/subscribe [::grab-focus?])
+             @(rf/subscribe [:page]))
+    (r/after-render
+     #(when-let [element (or (.querySelector js/document "h1")
+                             (.querySelector js/document "#main-content"))]
+        (focus/focus element)
+        (rf/dispatch-sync [::focus-grabbed])))))
+
+(defn- app []
+  (r/with-let [tracked (mapv r/track! [lazy-load-data! grab-focus!])]
+
+    (if (config/dev-environment?)
+      [react-strict-mode [page]]
```

Comment:
Are there a lot of warnings? Should browser tests fail if there are any/more?

Would actually be nice to fail any browser tests with any seen warnings (or maybe ignore some difficult to fix ones).

Reply:
Almost none actually, i have only seen one from `komponentit.autosize/textarea` for using (deprecated) react-dom/findDOMNode instead of ref. `react-transition-group` (not in this PR) also triggers that warning under strict mode if DOM element ref is not manually handled.

It could be good to test what warnings/how many are produced. #2368 already exists for tracking this, too.

Reply:
React 18 slightly changes component mounting/unmounting in strict mode: https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-strict-mode

Although this looks more related to using react/useState, which our components are not using (dropdown via react-select might be).